src/kit/Typography.tsx

```diff
@@ -0,0 +1,60 @@
+import { Typography } from 'antd';
+import { EllipsisConfig } from 'antd/es/typography/Base';
+import React from 'react';
+
+import { ValueOf } from 'kit/utils/types';
+
+import css from './Typography.module.scss';
+
+export const TypographySize = {
+  Default: 'default',
+  L: 'large',
```

Comment:
nit: the large and default sizes are very difficult to tell apart. i'd like us to make the large larger or the default smaller, maybe both
![Screenshot 2023-11-01 at 09-42-20 Standalone Design Kit](https://github.com/determined-ai/determined-ui/assets/15078396/56ff367f-6095-440f-9bdb-a28b628b5d85)

Reply:
decreasing Title Default and Small and increasing Body and Label Large, and tweaking the line heights to be divisible by 4px @michielpr